data/helpers/notesDb.js

```diff
@@ -0,0 +1,35 @@
+//grab knex configuration for accessing database
+const notesDb = require('../dbConfig.js');
+
```

Comment:
Great to see you using both helpers, and routers to streamline your codebase Christopher 🔥 I really like helpers b/c as things get more complex other small operations may pop up that you can easily create an additional helper for 👍